attestation-service/verifier/src/snp/mod.rs

```diff
@@ -347,17 +370,17 @@ mod tests {
         X509::from_der(&vcek).expect("failed to parse der");
 
         let cert_table = vec![CertTableEntry::new(CertType::VCEK, vcek)];
-        let VendorCertificates { ask, ark } = load_milan_cert_chain().as_ref().unwrap();
-        verify_cert_chain(&cert_table, ask, ark).unwrap_err();
+        let VendorCertificates { ask, ark, asvk } = load_milan_cert_chain().as_ref().unwrap();
+        verify_cert_chain(&cert_table, ask, ark, asvk).unwrap_err();
     }
 
     #[test]
     fn check_milan_chain_signature_failure() {
         let vcek = include_bytes!("test-vcek.der").to_vec();
         let cert_table = vec![CertTableEntry::new(CertType::VCEK, vcek)];
-        let VendorCertificates { ask, ark } = load_milan_cert_chain().as_ref().unwrap();
+        let VendorCertificates { ask, ark, asvk } = load_milan_cert_chain().as_ref().unwrap();
         // toggle ark <=> ask
-        verify_cert_chain(&cert_table, ark, ask).unwrap_err();
+        verify_cert_chain(&cert_table, ark, ask, asvk).unwrap_err();
```

Comment:
Regarding the tests, is there any chance you have a report generated with a vlek that we could use to create a new test?